models/migrations/v139.go

```diff
@@ -0,0 +1,20 @@
+// Copyright 2020 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package migrations
+
+import "xorm.io/xorm"
+
+func addAutoMergeTable(x *xorm.Engine) error {
+	type MergeStyle string
+	type ScheduledPullRequestMerge struct {
+		ID         int64      `xorm:"pk autoincr"`
+		PullID     int64      `xorm:"BIGINT"`
+		UserID     int64      `xorm:"BIGINT"`
+		MergeStyle MergeStyle `xorm:"varchar(50)"`
+		Message    string     `xorm:"TEXT"`
```

Comment:
```suggestion
		Message    string     `xorm:"TEXT"`
		CreatedUnix timeutil.TimeStamp `xorm:"created"`
```

maybe bug is here ...

Reply:
Oh nice catch.